java/src/org/openqa/selenium/grid/node/Node.java

```diff
@@ -116,13 +117,20 @@ public abstract class Node implements HasReadyState, Routable {
   protected final Tracer tracer;
   private final NodeId id;
   private final URI uri;
+  private final Duration sessionTimeout;
   private final Route routes;
   protected boolean draining;
 
   protected Node(Tracer tracer, NodeId id, URI uri, Secret registrationSecret) {
+    this(tracer, id, uri, registrationSecret, Duration.ofSeconds(300));
```

Comment:
Why do we need the default value for `sessionTimeout` here? Can't we change the signature? Does it break something?

Reply:
This constructor is used by 1 unit test, so I updated a specific value in the test alone and removed this constructor.